components/webdriver_server/actions.rs

```diff
@@ -176,7 +187,27 @@ impl Handler {
                     }
                 }
             },
-            ActionsType::Wheel { .. } => todo!("Not implemented."),
+            ActionsType::Wheel { actions } => {
+                for action in actions.iter() {
+                    match action {
+                        WheelActionItem::General(_action) => {
+                            self.dispatch_general_action(source_id)
+                        },
+                        WheelActionItem::Wheel(action) => {
+                            self.session_mut()
+                                .unwrap()
+                                .input_state_table
+                                .entry(source_id.to_string())
+                                .or_insert(InputSourceState::Wheel);
+                            match action {
+                                WheelAction::Scroll(action) => {
+                                    self.dispatch_scroll_action(action, tick_duration)?
+                                },
+                            }
```

Comment:
```suggestion
                            match action {
                                WheelAction::Scroll(action) => {
                                    self.dispatch_scroll_action(action, tick_duration)
                                },
                            };
```
Better make the coding style consistent with the other source types

Reply:
But will this return error if `self.dispatch_scroll_action` return error? Also, the `self.dispatch_pointermove_action` which return `Result<(), Errorstatus>` also use `?`

https://github.com/servo/servo/blob/main/components/webdriver_server/actions.rs#L166-L170

Reply:
Ack